packages/react-native-client/src/hooks/useAppScreenShare.ts

```diff
@@ -0,0 +1,71 @@
+import { useCallback, useState } from 'react';
+import RNFishjamClientModule from '../RNFishjamClientModule';
+import { SimulcastConfig } from '../types';
+import { ReceivableEvents, useFishjamEvent } from './useFishjamEvent';
+import { ScreenShareOptions } from './useScreenShare';
+import { Platform } from 'react-native';
+
+const defaultSimulcastConfig = () => ({
+  enabled: false,
+  activeEncodings: [],
+});
+
+let screenShareSimulcastConfig: SimulcastConfig = defaultSimulcastConfig();
+
+/**
+ * This hook can toggle client app screen sharing on/off and provides current screen share state. It works only on iOS.
+ * @returns An object with functions to manage app screen share on iOS and null on android.
+ * @category Screenshare
+ * @group Hooks
+ */
+export function useAppScreenShare(): {
+  isAppScreenShareOn: boolean;
+  simulcastConfig: SimulcastConfig;
+  toggleAppScreenShare: (
+    screenShareOptions?: Partial<ScreenShareOptions>,
+  ) => Promise<void>;
+} | null {
+  if (Platform.OS === 'android') return null;
```

Comment:
**issue:** 
```suggestion
  if (Platform.OS !== 'ios') return null;
```

What do you think about returning default values here instead of null? This way we would have a more pleasant API for the user, without having to unwrap nulls.